### PR TITLE
Health Economic Form

### PR DESCRIPTION
**Pull request by frdrckaman**

### CODE REVIEW

File: effect_subject/models/health_economics.py [hunk 1]
```diff
@@ -1,9 +1,262 @@
...
+    )
+
+    amount_spent_healthcare_last_month = models.IntegerField(
+        verbose_name="How much in total has been spent on your healthcare in the last month?"
+    )
+
+    activities_not_come_clinic = models.CharField(
+        verbose_name="What would you be doing in your life, if you had not come to the hospital or clinic",
+        max_length=45,
+        choices=CHILDCARE_CHOICES,
```
Comment (JonathanWillitts) [inline effect_subject/models/health_economics.py:77]: Childcare choices has NA option, which I don't think is applicable for this question.  I suggest creating a separate set of choices, with NA omitted.

**Review by JonathanWillitts — CHANGES_REQUESTED** [review]
Thanks Freddie,

I've made some suggestions to take a look at.  Please could you also?

- [x] update line 220 of `effect_subject\choices.py` to `Relative or others paying for drug`
- [ ] rename `health_economics.py` to `health_economics_event.py`
- [ ] rename `health_economics_baseline.py` to `health_economics.py`

File: effect_subject/models/health_economics_baseline_termination.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        verbose_name="How much does your household spend on rent (or house loan/mortgage) and utilities in a month?",
+    )
+
+    yearly_household_large_item_spent = models.IntegerField(
+        verbose_name="How much have you spent on large items (e.g., furniture, electrical items, cars) in the last "
+        "year? "
+    )
+
+    class Meta(CrfModelMixin.Meta, edc_models.BaseUuidModel.Meta):
+        verbose_name = "Health Economics Baseline"
```
Comment (JonathanWillitts) [inline effect_subject/models/health_economics_baseline_termination.py:84]:
```suggestion
        verbose_name = "Health Economics"
```

File: effect_lists/list_data.py [hunk 3]
```diff
@@ -314,6 +314,19 @@
...
+        ("ambulance", "Ambulance"),
+        ("private_taxi", "Private taxi"),
+        ("own_bicycle", "Own bicycle"),
+        ("hired_motorbike", "Hired motorbike"),
+        ("own_car", "Own car"),
+        ("own_motorbike", "Own motorbike"),
+        ("hired_bicycle", "Hired bicycle"),
+        ("foot", "Foot"),
+        (OTHER, "Other"),
+    ],
```
[the remark this answers is not in the document]
Comment (frdrckaman) [reply effect_lists/list_data.py:333]: I didnt write this, I just reuse it so my concern is when I rearrange it it my messup on other place where it has been used

**Review by JonathanWillitts — CHANGES_REQUESTED** [review]
Thanks for the prompt changes.

There are still a couple of things:

- [ ] the list data still needs sorting
- [ ] the renaming process doesn't seem to be finished.  i.e. the class names need renaming to match the (new) module names.  Everything should be either HealthEconomicsXyz, or HealthEconomicsEventXyz.  I've made a few suggestions to get you started 
- [ ] transport_used should be an m2m field
- [ ] can you make sure the tests are all passing in GitHub Actions (on your fork)?

File: effect_subject/admin/health_economics_admin.py [hunk 4]
```diff
@@ -2,17 +2,80 @@
...
-from ..forms import HealthEconomicsForm
 from ..models import HealthEconomics
 from .modeladmin import CrfModelAdmin
 
 
 @admin.register(HealthEconomics, site=effect_subject_admin)
 class HealthEconomicsAdmin(CrfModelAdmin):
 
-    form = HealthEconomicsForm
-
```
Comment (JonathanWillitts) [inline effect_subject/admin/health_economics_admin.py:14]: There needs to be an associated form for the HealthEconomics model

**Comment by JonathanWillitts** [comment]
Thanks Freddie, 

I'm merging this into a branch to review/make some final changes, with a view to including it into develop.  

I'll update you once it's finalised.

**Pull request merged**